ts/WoltLabSuite/Core/Acp/Ui/User/Action/ToggleConfirmEmailAction.ts

```diff
@@ -0,0 +1,60 @@
+/**
+ * @author  Joshua Ruesweg
+ * @copyright  2001-2021 WoltLab GmbH
+ * @license  GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
+ * @module  WoltLabSuite/Core/Acp/Ui/User/Action
+ * @since       5.5
+ */
+
+import AbstractUserAction from "./AbstractUserAction";
+import * as Ajax from "../../../../Ajax";
+import * as Core from "../../../../Core";
+import { AjaxCallbackSetup, DatabaseObjectActionResponse } from "../../../../Ajax/Data";
+import * as UiNotification from "../../../../Ui/Notification";
+
+export class ToggleConfirmEmailAction extends AbstractUserAction {
+  protected init(): void {
+    this.button.addEventListener("click", (event) => {
+      event.preventDefault();
+      const isEmailConfirmed = Core.stringToBool(this.userDataElement.dataset.emailConfirmed!);
+
+      Ajax.api(this, {
+        actionName: (isEmailConfirmed ? "un" : "") + "confirmEmail",
```

Comment:
```suggestion
        actionName: (isEmailConfirmed ? "unconfirmEmail" : "confirmEmail"),
```
Much more readable like this.